### PR TITLE
Move Fetcher frequency variable to top level

### PR DESCRIPTION
**Pull request by jeberl**
Summary
---------
Meant to do this before. Move the fetcher frequency variable from just the fetcher module to the to a top level tfvars variable

Test Plan
---------

`terraform validate && terraform apply`

### CODE REVIEW

File: hasher-matcher-actioner/terraform/fetcher/variables.tf [hunk 1]
```diff
@@ -47,10 +47,9 @@ variable "log_retention_in_days" {
   type        = number
 }
 
-variable "fetch_frequency_min" {
-  description = "How many minutes to wait between queries to ThreatExchange for updates"
-  type        = number
-  default     = 15
+variable "fetch_frequency" {
+  description = "How long to wait between calls to ThreatExcahnge. Must be an AWS Rate Expression. See here: https://docs.aws.amazon.com/AmazonCloudWatch/latest/events/ScheduledEvents.html"
+  type        = string
```
Comment (Dcallies) [inline hasher-matcher-actioner/terraform/fetcher/variables.tf:52]: I ran into problems with this a few times, thanks for changing!

**Review by Dcallies — APPROVED** [review]
Shipit

**Review by schatten — APPROVED** [review]
LGTM!

File: hasher-matcher-actioner/terraform/fetcher/variables.tf (hunk 1 above)
Comment (jeberl) [reply hasher-matcher-actioner/terraform/fetcher/variables.tf:52]: Seems like a terrible AWS design decision that you have to say `1 minute` but `5 minute**s**` lol

**Pull request merged**